tools/EMLassemblyline/make_eml.xml

```diff
@@ -1,134 +1,139 @@
+        <param name="desraster" type="text" label="Give a short description of your data file. If there is multiple data files please separate your descriptions with a comma (,) in the same order of your data input" optional = "true"/>
+        <param name="raster_url" type="text" label="Give the publicly accessible URL from which your raster data entity can be downloaded. If more than one, data files please separate your URLs with a comma (,) in the same order of your data input. If wanting to include URLs for some but not all then use a - for those that don't have a URL." optional = "true"/>
+      </section>
+      <section name="data_vector" title="Do you have vector data ? " >
+        <param name="datavector" type="data_collection"  collection_type="list"  label="Upload all data files wich are spatial vector" help= "Input a data collection." optional = "true"/>
+        <param name="desvector" type="text" label="Give a short description of your data file. If there is multiple data files please separate your descriptions with a comma (,) in the same order of your data input" optional = "true"/>
+        <param name="vector_url" type="text" label="Give the publicly accessible URL from which your vector data entity can be downloaded. If more than one, data files please separate your URLs with a comma (,) in the same order of your data input. If wanting to include URLs for some but not all then use a - for those that don't have a URL." optional = "true"/>
+      </section>
+      <section name="data_other" title="Do you have other data entity ? " >
+        <param name="dataother" type="data_collection"  collection_type="list"  label="Upload all data files wich are other entity" help= "Input a data collection." optional = "true"/>
+        <param name="desother" type="text" label="Give a short description of your data file. If there is multiple data files please separate your descriptions with a comma (,) in the same order of your data input" optional = "true"/>
+        <param name="other_url" type="text" label="Give the publicly accessible URL from which your other data entity can be downloaded. If more than one, data files please separate your URLs with a comma (,) in the same order of your data input. If wanting to include URLs for some but not all then use a - for those that don't have a URL." optional = "true"/>
+      </section>
+    </inputs>
+    <outputs>
+		 <data name="metadataout" from_work_dir="eml.xml" format="xml" label="EML"/>
+		 <data name="error" from_work_dir="err.txt" format="txt" label="Warning message"/>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="2"> 
+            <param name="templates" value="annotations.txt,keywords.txt,attributes_data_blary_al.txt,catvars_data_blary_al.txt,geographic_coverage.tabular,personnel.txt" >
+            </param>
+            <output name="metadataout" >

```

Comment:
can you please check if the indentation is 4 spaces? something is off here

Reply:
I think it is ok now! Thank you Björn!!